lib/manageiq/api/client/server_api.rb

```diff
@@ -0,0 +1,21 @@
+module ManageIQ
+  module API
+    module Client
+      class ServerApi
```

Comment:
This is an odd name for the class...Not sure if it should just be Api, or if you had a reason to name it this way.